frontend/src/App.less

```diff
@@ -1,12 +1,50 @@
+@import '../node_modules/bootstrap/less/variables.less';
+
 .app {
-  margin-top: 3em;
-  margin-bottom: 3em;
+    padding-top: 3em;
+    padding-bottom: 1em;
+    height: 100vh;
```

Comment:
maybe I'm too old school, but I would prefer

```css
html, body {
  margin: 0;
  padding: 0;
  height: 100%;
}

.app {
  height: 100%;
}
```

it's just easier to think about. Plus `vh` isn't accepted standart yet (even well supported across all browsers)